.evergreen.yml

```diff
@@ -712,6 +768,12 @@ buildvariants:
       # everything in one go
       - name: publish
 
+  - name: ubuntu_connectivity_tests
+    display_name: Connectivity Tests
+    run_on: ubuntu1604-large
```

Comment:
Since ubuntu 16.04 has already been EOL for almost a year and this is dockerized anyway, could we maybe use a newer distro like 20.04 here? I assume the build team will be interested in removing 16.04 support at some point

Reply:
sure